src/templates/_changelog.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="changelog-container nice-scrollbar">
```

Comment:
I have concerns about this markup, I think we could structure this a little better.  I'd like to see what this actually looks like first, though, before I suggest ripping this all apart.